index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="main.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="menu">
+            <h3>John Doe</h3>
+            <div><img src="./imgs/Union.svg" alt="hmb-button"></div>
+        </nav>
+        <div class="body-class">
+            <h1>Hey There. I’m Alex</h1>
+            <span>I’am a Software Developer</span>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+           <div class="social_media">
+            <img class="icon" alt="social_icon" src="imgs/gi.svg">
+            <img class="icon" alt="social_icon" src="imgs/in.svg">  
+            <img class="icon" alt="social_icon" src="imgs/Enabled.svg">
+            <img class="icon" alt="social_icon" src="imgs/Vector.svg">
+            <img class="icon" alt="social_icon" src="imgs/Disabled.svg">
+        </div>
+        </div>
+    </header>
+    
+</body>
```

Comment:
- [ ]  You should use the same spacing similar to the one from the Figma design.  I have highlighted the parts where the section should be improved, like the one from the Figma design. 

Figma design             |  Your design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/176643547-e338708d-8edb-4784-ac18-c5ae6cc60e74.png)  |  ![image](https://user-images.githubusercontent.com/43877689/176643384-6731c8c6-80dc-4c6d-baba-3a48cff6fd94.png)